src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -316,8 +316,17 @@ private void GroupFixes(Workspace workspace, IEnumerable<CodeFixCollection> fixC
                         {
                             if (fix.Action is SuppressionCodeAction)
                             {
-                                var suggestedAction = new SuppressionSuggestedAction(workspace, _subjectBuffer, _owner._editHandler,
-                                    fix, fixCollection.Provider, getFixAllSuggestedActionSet);
+                                SuggestedAction suggestedAction;
```

Comment:
Yes, I think Cyrus is keen on cleaning up our entire type hierarchy for code actions/suggested actions, which will probably simplify all this code.